server/authorization.go

```diff
@@ -131,6 +131,11 @@ func (a *AuthorizationServer) Check(ctx gocontext.Context, req *authv3.CheckRequ
 		}
 		log.Debugf("operation: %s", operation.Name)
 
+		// strip the basepath off path
```

Comment:
Nit: This could be moved to after the envRequest.IsAuthenticated() check below as it's not needed if that check fails.

Reply:
Sure.